app/src/main/java/edu/hm/cs/ma/todoguru/task/read/ViewTaskFragment.kt

```diff
@@ -10,15 +10,16 @@ import androidx.lifecycle.ViewModelProvider
 import androidx.navigation.fragment.navArgs
 import edu.hm.cs.ma.todoguru.R
 import edu.hm.cs.ma.todoguru.database.ToDoGuruDatabase
-import edu.hm.cs.ma.todoguru.databinding.ViewTaskFragmentBinding
+import edu.hm.cs.ma.todoguru.databinding.ViewTaskFragmentBindingImpl
 import edu.hm.cs.ma.todoguru.task.timeTracker.TimeTrackerViewModel
+import kotlinx.android.synthetic.main.view_task_fragment.*
 import kotlinx.android.synthetic.main.view_task_fragment.description_task
 import kotlinx.android.synthetic.main.view_task_fragment.dueDate_task
 import kotlinx.android.synthetic.main.view_task_fragment.title_task
 
 class ViewTaskFragment : Fragment() {
     private val args: ViewTaskFragmentArgs by navArgs()
-    private lateinit var binding: ViewTaskFragmentBinding
+    private lateinit var binding: ViewTaskFragmentBindingImpl
```

Comment:
use always the interface like `ViewTaskFragmentBinding` instead of the implementation like `ViewTaskFragmentBindingImpl`